client/landing/stepper/hooks/use-processing-loading-messages.ts

```diff
@@ -0,0 +1,48 @@
+import { Onboard } from '@automattic/data-stores';
+import { useSelect } from '@wordpress/data';
+import { useI18n } from '@wordpress/react-i18n';
+import { STEPPER_INTERNAL_STORE } from 'calypso/landing/stepper/stores';
+
+const SiteIntent = Onboard.SiteIntent;
+
+export function useProcessingLoadingMessages() {
```

Comment:
Since this is specific to the processing step, we can put this hook in `client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks` instead of `client/landing/stepper/hooks`

Reply:
Create new folder /hooks in /processing-step should be good?

Reply:
I moved it to [/processing-step/hooks](https://github.com/Automattic/wp-calypso/pull/65456/commits/fa2b94e153e327e24747c5ab1f5f044382f7509f)